spec/vc-11.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "definitions": {
+    "credentialStatus": {
+          "type": "object",
+          "properties": {
+            "id": { "type": "string" },
+            "type": { "type": "string" },
+            "statusPurpose": { "type": "string" },
+            "statusListIndex": { "type": "string" },
+            "statusListCredential": { "type": "string" }
+          },
+          "required": ["id", "type", "statusPurpose", "statusListIndex", "statusListCredential"]
+        },
+        "credentialSchema": {
+          "type": "object",
+          "properties": {
+            "id": { "type": "string" },
+            "type": { "type": "string" }
+          },
+          "required": ["id", "type"]
+        }
+  },
+  "properties": {
+    "@context": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1
+    },
+    "id": {
+      "type": "string"
+    },
+    "type": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1
+    },
+    "issuer": {
+      "oneOf": [
+        { "type": "string" },
+        {
+          "type": "object",
+          "properties": {
+            "id": { "type": "string" },
+            "name": { "type": "string" }
+          },
+          "required": ["id"]
+        }
+      ]
+    },
+    "issuanceDate": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "expirationDate": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "credentialSubject": {
```

Comment:
<img width="868" alt="image" src="https://github.com/TBD54566975/web5-spec/assets/5314059/34ff1523-b4fa-4c2b-adb9-7ca1d375ecd2">

It `may` have an id, will we accept an external vc without and id?

Reply:
I will add this good spot

Reply:
the doc here specifies it must be present https://github.com/TBD54566975/web5-spec/blob/main/spec/vc.md#verifiable-credential-data-model

Reply:
we could certainly accept creds without em but we shouldn't produce creds without ids